resolwe_bio/tests/processes/test_reads_filtering.py

```diff
@@ -345,67 +345,47 @@ def test_bbduk_single(self):
             inputs = {'src': ['bbduk test reads.fastq.gz']}
             reads = self.run_processor('upload-fastq-single', inputs)
 
-            inputs = {'src': 'bbduk_adapters.fasta'}
-            adapters = self.run_processor('upload-fasta-nucl', inputs)
-
         inputs = {
             'reads': reads.id,
-            'adapters': {
-                'reference': [adapters.id]},
-            'trimming_par': {
-                'kmask': 'lc'},
-            'barcode_par': {
-                'barcode_seq': ['GCGTTT', 'TATGNNN']}
         }
-
         filtered_reads = self.run_process('bbduk-single', inputs)
-        self.assertFiles(filtered_reads, 'fastq', ['filtered_reads_bbduk_single.fastq.gz'], compression='gzip')
-        del filtered_reads.output['fastqc_url'][0]['total_size']  # Non-deterministic output.
-        self.assertFields(filtered_reads, "fastqc_url", [{'file': 'fastqc/bbduk test reads_fastqc/fastqc_report.html',
-                                                          'refs': ['fastqc/bbduk test reads_fastqc'],
-                                                          'size': 303594}])
 
-        inputs = {
-            'reads': reads.id,
-            'adapters': {
-                'bbduk_adapters': 'truseq'},
-            'trimming_par': {
-                'kmask': 'lc'},
-            'barcode_par': {
-                'barcode_seq': ['GCGTTT', 'TATGNNN']}
+        self.assertFiles(filtered_reads, 'fastq', ['bbduk_reads.fastq.gz'], compression='gzip')
```

Comment:
Please retain the whitespace in the file name. Maybe also split the reads into two files and upload them as two separate seq lanes to cover this part of the process code.

Reply:
This is an output test assertion. If it doesn't handle filenames with spaces correctly, then we have much bigger problems :wink:. The input file is defined a few lines above and already contains a space. I'll follow your suggestion and use two input files though.

Reply:
:+1: